src/lib/dashboard-model.js

```diff
@@ -3,17 +3,22 @@
 const config = require('../config');
 const query = config.postgres.mock ? require('../../mock/mock-query') : require('./db/query');
 
-const costSaving = () => query.totalCustomerSearches()
-  .then(searches => Math.round(((searches / 2) + 793235) * 1.81));
+// Martin's Multiplier, an estimated ratio of relevant searches in the audit table, currently 55%
+const M = 0.55;
```

Comment:
It would be nice to including something in the commit message on how we calculated this. (Assuming it is publishable! - If not perhaps we could reference a private document?)

Reply:
(or JIRA.)

Reply:
This is already linked with Jira, I thought about linking the Jira issue, but it didn't seem to add much and wouldn't be accessible to most github users